src/composables/transactionList.ts

```diff
@@ -58,18 +61,17 @@ function updateAccountTransaction(address: Encoded.AccountAddress, transaction:
   }
 }
 
-const setTransactionsNextPage = (address: Encoded.AccountAddress, url: string) => {
+const setTransactionsNextPage = (address: string, url: string | null) => {
```

Comment:
```suggestion
const setTransactionsNextPage = (address: string, url?: string) => {
```

Reply:
null is returned from api if we get last page.